modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/SnapshotRestoreProcess.java

```diff
@@ -621,10 +620,20 @@ private IgniteInternalFuture<SnapshotRestoreOperationResponse> prepare(SnapshotO
      * @param cacheDir Cache directory.
      * @return Temporary directory.
      */
-    private static File formatTmpDirName(File cacheDir) {
+    static File formatTmpDirName(File cacheDir) {
         return new File(cacheDir.getParent(), TMP_CACHE_DIR_PREFIX + cacheDir.getName());
     }
 
+    /**
+     * @param tmpCacheDir Temporary cache directory.
+     * @return Cache or group id.
+     */
+    static int tmpDirGroupId(File tmpCacheDir) {
```

Comment:
It seems the naming is not correct. We are getting the groupId here, however, it is not temporary. I think it is better to consider the following names: `groupIdFromTmpDir` or just `restoredCacheId`

Reply:
Renamed